packages/nuejs.org/docs/page-layout.md

```diff
@@ -1,10 +1,10 @@
 
 # Page layout
-Global design system standardizes the structure of your web pages so you can use the same HTML markup, but vary your external CSS to achieve wildly different designs.
+A global design system standardizes the structure of your web pages, allowing you to use the same HTML markup while applying different external CSS to achieve a wide range of designs.
```

Comment:
```suggestion
A global design system standardizes the structure of your web pages, allowing you to use the same HTML markup, while applying different external CSS, to achieve a wide range of designs.
```

That's how I would read it, feel free to reject

Reply:
👍